hooks/pre_prompt.py

```diff
@@ -3,10 +3,11 @@
 from ccds.version import __version__ as version
 
 if __name__ == "__main__":
-    if version <= "2.0.0":
+    if version < "2.0.1":
         warnings.warn(
             "It looks like you're using a version of CCDS that "
-            "always uses the latest template. For more stable behavior, "
-            "update to a newer version of CCDS.",
+            "defaults to using the latest template. For more stable behavior, "
+            "update to CCDS version 2.0.1 or later. To upgrade to the latest "
+            "version, run 'pip install -U cookiecutter-data-science'",
```

Comment:
I might say "Upgrade to the latest with your package manager, for example `pip install -U cookiecutter-data-science`."